.github/workflows/release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build and push
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Install dependencies
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
+      - name: Call linter
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: lint
+      - name: Build code
+        uses: borales/actions-yarn@v2.0.0
+        env:
+          NODE_OPTIONS: '--max-old-space-size=8192'
+        with:
+          cmd: build
+      - name: Get tag
+        id: get_tag
+        run: echo ::set-env name=GIT_TAG::${GITHUB_REF/refs\/tags\//}
```

Comment:
How is GITHUB_REF populated?

Reply:
I see now... https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
Provided by GitHub: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables

Reply:
Uou can see it in action here (created a tag just for testing):
https://github.com/eranco74/assisted-installer/actions/runs/136882098